agent/bench-scripts/pbench-run-benchmark.pl

```diff
@@ -143,13 +147,15 @@ sub find_common_parameters {
     }
 }
 
+my $date = get_pbench_datetime();
+
 # Prepare all the dirs for the run
 if ($pp_only) {
     if (! -e $base_bench_dir) {
         die "Expected the base benchmark directory, [$base_bench_dir], to already exist, since this is a post-process only mode";
     }
 } else {
-    $base_bench_dir = get_benchmark_results_dir($benchmark, $params{"user-tags"});
+    $base_bench_dir = get_benchmark_results_dir($benchmark, $params{"user-tags"}, $date);
```

Comment:
This correct the date used in the name of the directory to match what is stored as the date value in the `metadata.log` file.